test/Blocks/test_analysis_points.jl

```diff
@@ -165,7 +166,7 @@ function SystemModel(u = nothing; name = :model)
            connect(inertia2.flange_a, spring.flange_b, damper.flange_b)]
     if u !== nothing
         push!(eqs, connect(torque.tau, u.output))
-        return @named model = ODESystem(eqs, t;
+        return ODESystem(eqs, t;
```

Comment:
then the name needs to be passed

Reply:
Resolved here: https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/274/commits/eae592ba86e9de11fa575f005d3c13b526a80c99#diff-3603e44df7347f59dcb8354b6aa1af55ec51d86974ae7a5521ae7214cba54c27R177-R178